src/components/Button.tsx

```diff
@@ -6,12 +6,12 @@ export class Button extends React.Component<{
   title?: string;
   isDisabled?: boolean;
   onClick?: Function;
-  customClass?: string;
+  customClassName?: string;
 }, {}> {
   render() {
     let className = "button";
-    if(this.props.customClass){
-        className = this.props.customClass
+    if(this.props.customClassName){
```

Comment:
Formatting nit, space after `if`, also before the brace. Now that we just landed tslint  this will be automatically detected in the future.